Nodejs/Product/InteractiveWindow/Theme/Microsoft.NodejsTools.InteractiveWindow.theme.v14.0.pkgdef

```diff
@@ -1,12 +0,0 @@
-[$RootKey$\Themes\{a4d6a176-b948-4b29-8c66-53c97a1ed7d0}\NodejsToolsInteractive]
```

Comment:
I assume all these pkgdef and vstheme files are no longer needed then? Due to this change? Or just cleaning up?